package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vusd-dapp",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
As this is not a package, we don't actually need to bump the version. So usually our websites have the same version always 😅

Reply:
Yeah. No need to do this for VUSD. Doesn't hurt either.